src/Case-Channel.f90

```diff
@@ -190,6 +206,23 @@ subroutine boundary_conditions_channel (ux,uy,uz,phi)
        endif
     endif
 
+    if( mhd_active .and. iimplicit<=0 .and. mhd_equation ) then
+       ! FIXME add a test
+       ! This is valid only when nclyB*1 = 2
+       if (xstart(2) == 1) then
+          Bm(:,1,:,1)  = zero
+          Bm(:,1,:,2)  = zero
+          Bm(:,1,:,3)  = zero
+       endif
+       ! FIXME add a test
```

Comment:
this fixme is a minor issue

Reply:
I have removed the implicit time scheme for the MHD and Channel since it was not used. If this is required for some specific cases we'll add this back in

Reply:
I'll remove this

Reply:
The boundary conditions at this location are for the explicit time scheme. I think this should not be removed.

Reply:
Yes I am keeping it.

Reply:
The boundary conditions for the implicit time scheme are around line 666 in implicit.f90. See https://github.com/xcompact3d/Incompact3d/blob/master/src/implicit.f90#L666

Reply:
Don't understand the FIXME with add a test

Reply:
The line above is `if ((nclyS1 == 2).and.(xstart(2) == 1)) then`. I think for the magnetic field, it should be similar and we should impose the value at the boundary only when nclyB*** is set to 2